vitrina/requests/views.py

```diff
@@ -14,3 +17,47 @@ class RequestListView(ListView):
 class RequestDetailView(DetailView):
     model = Request
     template_name = 'vitrina/requests/detail.html'
+
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        request = context_data.get('request')
```

Comment:
```suggestion
        request: Request = self.object
```
Please annotate data types, it took me some time to figure out, what exactly is `request`. Type annotations helps to better understand the code.